linchpin/cli/__init__.py

```diff
@@ -138,7 +138,18 @@ def _write_to_inventory(self, tx_id=None, inv_path=None, inv_format="cfg"):
                         lt_data = targets[name]["inputs"]["layout_data"]
                         t_data = targets[name]["inputs"]["topology_data"]
                         layout = lt_data["inventory_layout"]
-                        i_path = targets[name]["outputs"]["inventory_path"][0]
+                        # check whether inventory_file is mentioned in layout
+                        if layout.get("inventory_file", None):
+                            i_path = layout["inventory_file"]
+                            # if its not absolute path make path
+                            # relative to workspace
+                            if not os.path.isabs(i_path):
+                                i_path = "{0}/{1}".format(self.ctx.workspace,
+                                                          i_path)
```

Comment:
I think we need to make this use the configuration settings in linchpin.conf, not just `workspace/inventory_path`.  See https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/linchpin/linchpin.constants#L54

Reply:
@samvarankashyap Good catch on that one. Maybe we should scan the rest of the code and see where 'default_inventories_path' is used? I'm thinking it's not used much outside of the playbooks, if at all. We could adjust it to use a better path. 

BTW, the 'defaults' bit is probably wrong anyway. :smile: